psi4/driver/procrouting/dft/hyb_functionals.py

```diff
@@ -534,7 +534,7 @@ def get_pw6b95_tweaks():
     },
     "dispersion": {
         "nlc": False,
-        "type": "d3bj",
+        "type": "d3bj2b",
```

Comment:
The ATM default for classic dftd3 is off and s-dftd3 is on, so that's what a raw qcengine call would do since qcengine's mandate (approximately) is pass-through data rearrangement. Psi4's goal is an even interface over all the disp programs, so it pre-solves the parameters and makes a customized request to either qcengine harness. That way, the Psi4 default is 2-body, no matter whether dftd3 or s-dftd3 is available. If one requests b3lyp-d3bjatm and only classic dftd3 is around, it'll balk. Somewhere I'll need to document that better.

Reply:
Reading this again, I think I understand the question. If the user had an input with d3bj, no need to edit, as d3bj did and now continues to default to 2body-only, regardless of engine. I changed the type here (1) to emphasize that d3bj2b is the fundamental type rather than d3bj and (2) these definitions, as opposed to input files, don't get aliases applied so they need the fundamental type (possibly I relaxed this).

Reply:
I asked because of your point (2). We developers then need to pay attention to the correct type.

However users with old input files might get slightly different results if they use a custom functional. Likely a very very small number of users. Those probably read the changelog.

okay to not relax this.

Reply:
I've checked with the dft-custom test, and one can use the aliases in custom functionals, so old inputs will work and answers not change. Apparently the fundamental type requirement was an earlier version of the code.